lang/talon/talon.talon

```diff
@@ -17,9 +17,7 @@ tag(): user.code_functions_gui
 dot talon: insert(".talon")
 #defintion blocks for the context
 action block:
```

Comment:
Heh, we can drop this command now right?

Reply:
Yeah, that's an issue for a separate commit/PR though.